test/liquidation/liquidation.test.js

```diff
@@ -95,4 +88,31 @@ describe("Integration -- liquidation happy path", () => {
       )
     })
   })
+
+  describe("when deposit has changed the state outside the coverage pool during pending coverage pool auction", () => {
+    let auction
+    beforeEach(async () => {
+      auction = await prepareAuction()
+      // simulate deposit state change outside Coverage Pools
+      await tbtcDeposit.setStateLiquidated()
```

Comment:
Instead of introducing a separate, test-only function, can we call `purchaseSignerBondsAtAuction()`? This is what actually happens in this case: third party purchasing signer bonds directly from tBTC deposit.

Reply:
Fixed in 2c29566b2fba6d7c26e266b118d46d24f18e4d5c.